test/e2e/gui/objects_map/settings_names.py

```diff
@@ -15,7 +15,7 @@
 mainWindow_scrollView_StatusScrollView = {"container": mainWindow_LeftTabView, "id": "scrollView", "type": "StatusScrollView", "unnamed": 1, "visible": True}
 scrollView_MenuItem_StatusNavigationListItem = {"container": mainWindow_scrollView_StatusScrollView, "type": "StatusNavigationListItem", "visible": True}
 scrollView_Flickable = {"container": mainWindow_scrollView_StatusScrollView, "type": "Flickable", "unnamed": 1, "visible": True}
-settingsBackUpSeedPhraseOption = {"container": mainWindow_scrollView_StatusScrollView, "objectName": "19-MainMenuItem", "type": "StatusNavigationListItem", "visible": True}
+settingsBackUpSeedPhraseOption = {"container": mainWindow_scrollView_StatusScrollView, "objectName": "18-MainMenuItem", "type": "StatusNavigationListItem", "visible": True}
```

Comment:
This could use some constant @anastasiyaig, instead of hardcoding what we do in `Constants.qml`

Reply:
i dont understand @caybro what do you mean?

Reply:
Basically using sth like `Constants.settingsSubsection.backUpSeed` instead of the literal `18`

Reply:
got it, will try to find out why we rely on real names now in tests and if that could be improved, thanks for hint